tests/infra/github.py

```diff
@@ -74,27 +75,38 @@ def get_major_version_from_branch_name(branch_name):
     )
 
 
+def get_debian_package_url_from_tag_name(tag_name):
+    return f'{REMOTE_URL}/releases/download/{tag_name}/{tag_name.replace("-", "_")}{DEBIAN_PACKAGE_EXTENSION}'
+
+
+def has_release_for_tag_name(tag):
+    return requests.get(f"{REMOTE_URL}/releases/tag/{tag}").status_code == 200
```

Comment:
We could .head() on the release binary directly to check if it exists or not. It's nice if the release is there, but ultimately what we want is the .deb.

Reply:
Good point. This simplifies things slightly too!